modules/Configuration.py

```diff
@@ -73,9 +74,41 @@ def get_coin_cfg():
                                         maxtolendrate=(Decimal(cur[5])) / 100)
         except Exception as ex:
             print "Coinconfig parsed incorrectly, please refer to the documentation. Error: " + str(ex)
+    else:
+        for cur in FULL_LIST:
+            if config.has_section(cur):
+                try:
+                    from decimal import Decimal
+                    coin_cfg[cur] = {}
+                    coin_cfg[cur]['minrate'] = (Decimal(config.get(cur, 'mindailyrate'))) / 100
+                    coin_cfg[cur]['maxactive'] = Decimal(config.get(cur, 'maxactiveamount'))
+                    coin_cfg[cur]['maxtolend'] = Decimal(config.get(cur, 'maxtolend'))
+                    coin_cfg[cur]['maxpercenttolend'] = (Decimal(config.get(cur, 'maxpercenttolend'))) / 100
+                    coin_cfg[cur]['maxtolendrate'] = (Decimal(config.get(cur, 'maxtolendrate'))) / 100
+                except Exception as ex:
+                    print "Coinconfig for " + cur + " parsed incorrectly, please refer to the documentation. " + \
+                          "Error: " + str(ex)
+                    # Need to raise this exception otherwise the bot will continue if you configured one cur correctly
+                    raise
     return coin_cfg
 
 
+def get_min_loan_sizes():
+    min_loan_sizes = {}
+    for cur in FULL_LIST:
+        if config.has_section(cur):
+            try:
+                min_loan_sizes[cur] = config.get(cur, 'minloansize')
```

Comment:
Fixed all the doc updates, I don't get the last fix to Configuration.py though. Casting to Decimal is fine. But, I'm not sure where you mean to use get(). I'm using it for getting the minloansize already I thought?